instrumentation/spring-webflux-5.1.0/src/main/java/org/springframework/http/ReactiveHttpOutputMessage_Instrumentation.java

```diff
@@ -23,32 +23,6 @@
     @NewField
     public Token token;
 
-    public Mono<Void> writeWith(Publisher<? extends DataBuffer> body) {
-        try {
-            if (this.token != null) {
-                this.token.expire();
-                this.token = null;
-            }
-        } catch (Throwable t) {
-            AgentBridge.instrumentation.noticeInstrumentationError(t, Weaver.getImplementationTitle());
-        }
-
-        return Weaver.callOriginal();
-    }
-
-    public Mono<Void> writeAndFlushWith(
-            Publisher<? extends Publisher<? extends DataBuffer>> body) {
-        try {
-            if (this.token != null) {
-                this.token.expire();
-                this.token = null;
-            }
-        } catch (Throwable t) {
-            AgentBridge.instrumentation.noticeInstrumentationError(t, Weaver.getImplementationTitle());
-        }
-        return Weaver.callOriginal();
-    }
-
```

Comment:
This is the expiring of tokens you said was being done defensively, but which not actually needed, correct?

Reply:
Yes, it turns out that the writeWith and writeAndFlushWith can get called during the reactive flow which prevents further operations from linking the token and effectively ending the Transaction early.